lib/awoo.js

```diff
@@ -56,22 +69,22 @@ async function awooCore (site, fn) {
 
   debug('awoo is ready to go!!!')
 
-  const resSite = await fn(site)
-  if (Object.keys(site._config).length === 0) {
-    site._config = generateConfig({})
+  const finalSite = await fn(site) || site
+  if (Object.keys(finalSite._config).length === 0) {
+    finalSite._config = generateConfig({})
   }
-  return buildSite(resSite)
+  return buildSite(finalSite)
 }
 
 async function awoo (fn) {
-  const site = {}
+  var site = {}
```

Comment:
Was something about this wrong? Assigning properties doesn't need a `var` statement, right?

Reply:
Oh, I think this'll be fine.